### ISSUE TITLE
minimum file does not exist

### ISSUE THREAD

**Issue by EvanCarroll** [haikarainen/light#115]
I'm getting a weird error when I try to set my micmute light

```
$ light -S 1 -s "sysfs/leds/platform::micmute" -v 3
helpers.c:21: Error: could not open '/etc/light/targets/sysfs/leds/platform::micmute/minimum' for reading
helpers.c:21: Error: Verify it exists with the right permissions
```

Here are the files in sys,

```
find ./devices/platform/thinkpad_acpi/leds/platform::micmute/
./devices/platform/thinkpad_acpi/leds/platform::micmute/
./devices/platform/thinkpad_acpi/leds/platform::micmute/uevent
./devices/platform/thinkpad_acpi/leds/platform::micmute/brightness
./devices/platform/thinkpad_acpi/leds/platform::micmute/power
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/runtime_active_time
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/runtime_active_kids
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/runtime_usage
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/runtime_status
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/autosuspend_delay_ms
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/async
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/runtime_suspended_time
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/runtime_enabled
./devices/platform/thinkpad_acpi/leds/platform::micmute/power/control
./devices/platform/thinkpad_acpi/leds/platform::micmute/device
./devices/platform/thinkpad_acpi/leds/platform::micmute/trigger
./devices/platform/thinkpad_acpi/leds/platform::micmute/subsystem
./devices/platform/thinkpad_acpi/leds/platform::micmute/max_brightness
```

**Comment by jlevers** [comment]
Not sure that this is the same issue, but have you tried the fix suggested in #96?

**Comment by haikarainen** [comment]
This is a non-issue. You will always get this error if you dont have a minimum file, which is not required. The real issue here is that we report it as an error. This happened when someone wanted to "clean up" how we reported file IO errors...

You wont see the error without the verbosity flag. If the light doesnt change with your command, the controller probably doesnt work or might be only implemented as readonly.
